docs/event-processing/dead-letter-stream.md

```diff
@@ -6,7 +6,6 @@ seo:
 
 # Dead Letter Stream
 
-<!-- TODO: Event Processing Application is a pattern that should be linked to in the following text -->
 [Event Processing Applications](event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
```

Comment:
```suggestion
[Event Processing Applications](../event-processing/event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
```

Reply:
It's in the same folder, the link works as is, i think the extra characters aren't necessary